src/client/AM.PA.MonitoringTool/WindowsActivityTracker/Visualizations/DayFragmentationTimeline.cs

```diff
@@ -6,11 +6,13 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection;
```

Comment:
I only have minimal C# experience, but adding a `using`-statement without changing the code seems incorrect. Is this import really used?

Reply:
You're absolutely right. This stems from a code change I didn't push in the end. My bad.